gravitee-apim-console-webui/src/management/application/details/general/application-general.component.html

```diff
@@ -153,5 +153,19 @@ <h3>OpenID Connect Integration</h3>
     </mat-card>
   </ng-container>
 
+  <mat-card class="danger-card">
```

Comment:
If the user cannot perform any actions in the danger zone, the card should not be shown.

Reply:
Done